src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,274 @@
+    onMouseOut: () => debouncedSetTooltipVisibility(false),
+    onClick: () => setModalVisibility(true),
+  }
+
+  const popperProps = {
+    placement,
+    outOfBoundaries: true,
+    modifiers,
+    eventsEnabled: true,
+  }
+
+  const tooltipClasses = [styles.root, inline ? styles.inline : styles.block]
+
+  const renderTooltip = (
+    <div className={tooltipClasses.join(' ')}>
+      <Manager>
+        <Reference>
+          {({ ref }) =>
+            children ? (
+              <div ref={ref} {...referenceProps}>
+                {children}
+              </div>
+            ) : (
+              <div className={styles.icon} ref={ref} {...referenceProps}>
+                {tooltipVisible ? Tooltip.SVGS.iconHover : Tooltip.SVGS.icon}
+              </div>
+            )
+          }
+        </Reference>
+        <div className={styles.popperContainer}>
+          <Popper {...popperProps}>
+            {({ ref, ...rest }) => (
+              <PopperContent
+                innerRef={ref}
+                visible={tooltipVisible}
+                details={details}
+                {...rest}
+              />
+            )}
+          </Popper>
+        </div>
+      </Manager>
+    </div>
+  )
+
+  return (
+    <>
+      {isMobile() && renderModal}
+      {renderTooltip}
+    </>
+  )
+}
+
+const PopperContent = ({
+  innerRef,
+  visible,
+  style,
+  placement,
+  arrowProps,
+  scheduleUpdate,
+  details,
+}) => {
+  const [isPositioned, setIsPositioned] = useState(false)
+
+  const prevVisible = usePrevious(visible)
+  const prevPosition = usePrevious(style.transform)
+  const position = style.transform || ''
+
+  const debouncedScheduleUpdate = useRef(
+    debounce(
+      () => {
+        scheduleUpdate()
+      },
+      1000,
+      { trailing: true }
+    )
+  ).current
+
+  useEffect(() => {
+    window.addEventListener('scroll', debouncedScheduleUpdate)
+    return () => {
+      window.removeEventListener('scroll', debouncedScheduleUpdate)
+    }
+  }, [position])
+
+  const contentBoxClasses = [
+    styles.contentBox,
+    visible && isPositioned ? styles.visible : styles.hidden,
+  ]
+
+  // On first reveal of tooltip, schedule an update so positioning
+  // is correct incase the DOM has shuffled since the page first loaded
+  if (prevVisible === false && visible === true && !isPositioned) {
+    setIsPositioned(true)
+    scheduleUpdate()
+    return
+  }
+
+  // Check Tooltip Positioning, remove transition if repositioning is necessary
+  if (prevPosition !== position) {
+    contentBoxClasses.push(styles.noTransition)
+  }
+
+  return (
+    <div className={contentBoxClasses.join(' ')} ref={innerRef} style={style}>
+      <Footnote.Regular400>{details}</Footnote.Regular400>
+      <div
+        ref={arrowProps.ref}
+        className={styles.arrow}
+        style={arrowProps.style}
+        data-placement={placement}
+      />
+    </div>
+  )
+}
+
+Tooltip.SVGS = {
+  closeButton: (
+    <svg
+      width="14"
```

Comment:
Nit: I usually prefer to set this via CSS ... but we don't have a proper SVG pipeline 🤦‍♂ so not your fault/issue here